origin-messaging/src/entry.js

```diff
@@ -0,0 +1,204 @@
+'use strict'
+
+const Clock = require('./lamport-clock')
+const isDefined = require('./is-defined')
+
+const IpfsNotDefinedError = () => new Error('Ipfs instance not defined')
+
+class Entry {
+  /**
+   * Create an Entry
```

Comment:
OMG you are heavily commenting your code ? I have never seen this before... I love it ❤️

Reply:
This is copied code from orbit....

Reply:
ha. I should have known better... sad panda...